### DISCUSSION TITLE
1.0.0 신규 버전에 추가할 신기능을 제안 받습니다!

### DISCUSSION THREAD

**Discussion by note11g** [note11g/flutter_naver_map#25] [Ideas]
flutter_naver_map이 1.0.0 버전으로 리뉴얼 예정입니다.

**사용하시면서 불편하셨던 점, 있으면 좋을 것 같은 기능들을 마구 제안해주세요!**

패키지의 방향성과 맞고, 실현가능하다고 여겨지는 것들은 탑재하도록 하겠습니다! 감사합니다!!!

* 해당 게시글에 남겨주시면 됩니다.

**Comment by RyanAdOS** [comment]
마커를 단일이미지가 아닌, 위젯형태로 넣어보고싶어요ㅎ

**Comment by rogue1228** [comment]
구글 지도의 마커 클러스터 기능 추가 하고 싶은데 계획이 있으신가요?

**Reply by rogue1228** [reply]
추가로 js 플랫폼 추가는 계획 있으신지 궁금합니다.

**Reply by note11g** [reply]
JS 플랫폼은 완전히 다른 베이스를 갖고 있어서 현재로써는 계획에 없는 점, 참고하시기 바랍니다. 감사합니다.

**Comment by rambozzang** [comment]
마커의 움직임이 애니메이션 이동이 가능했으면 합니다.

**Comment by stellapath** [comment]
네트워크 이미지 또는 바이트 형식의 이미지를 마커에 적용할 수 있으면 좋을 것 같습니다.

현재 개인적으로 수정하여 UInt8List 형식의 이미지를 각각 Android-Bitmap, iOS-UIImage 로 변환해서 마커에 적용하고 있거든요..

**Reply by ItdaTeam** [reply]
마커 클러스터링 기능 꼭 필요 합니다.!!

**Comment by rambozzang** [comment]
1.0 언제 출시되나요? 그리고 포함되는 기능들도 리스트업 부탁드립니다.

**Reply by note11g** [reply]
죄송합니다. 시험 이슈로 계속 딜레이되고 있네요.. core 개발이 아직 끝나지 않아 core 개발이 끝나면 readme 업데이트 예정입니다. 감사합니다.

**Reply by hismethod** [reply]
개발중인 브랜치를 올려주시는건 어떠신가요? 그럼 PR도 활발해지고 좋을것 같아요.

**Comment by note11g** [comment]
** 1.0 업데이트는 2022년 11월 20일 이후에 제공됩니다 **
수능 이슈로 미뤄진 점, 사과 말씀 드립니다.
기존 버전에 대한 PR / 버그 수정은 11월 20일 이전에도 계속 이루어질 예정입니다. 감사합니다.

**Reply by eastswift** [reply]
지금 basemap 소스를 보면. mylocation 버튼이 주석 처리 되어있습니다. 이것을 작동 됐으면 좋겠습니다!

**Comment by ItdaTeam** [comment]
클러스터링 꼭되었음합니다

2022년 11월 11일 (금) 오전 9:41, eastswift ***@***.***>님이 작성:

**Comment by eastswift** [comment]
너무 좋은 라이브러리라 정말 감사하게 잘 쓰고있습니다

**Comment by hdjeong80** [comment]
혹시 이거 진행중이세요? 아니면 캔슬인가요? 지금 이 패키지는 최신 플러터에서는 동작을 안하는 상태인거죠?

**Reply by note11g** [reply]
진행중이며, 현재 버전 역시 최신 플러터에서 동작함을 확인하였습니다.
추가로, 새로운 1.0.0 버전은 플러터 3.0.0이 강제됩니다.

**Reply by note11g** [reply]
딜레이되는 이유 중에 하나는, 1.0.0에서는 새로운 PlatformView 방식을 사용하는데, android에서 view가 정확한 위치에 attatch 되지 않는 현상을 수정하려고 시도하는 중이기 때문입니다.

**Reply by note11g** [reply]
오늘 점심에 dev_1.0 브랜치에 1.0.0 개발중인 코드 업로드 되었습니다.

Android의 경우, 바로 사용하실 수 있습니다. 감사합니다.

**Comment by OmniTech1** [comment]
코린이의 입장에서 DOCS와 클러스터링 존버하고 있습니다. 개발에 감사드립니다.

**Comment by jsk005** [comment]
Android Naver Map 라이브러리도 네이버가 공식적으로 클러스터링 기능을 지원하지 않았습니다.
박상권님이 만든 클러스터링 라이브러로 커스텀 마커를 모아보기 제공으로 해결하여 편리하였습니다.
Flutter Naver Map 라이브러리를 개발해 주셔서 감사드립니다.
1. 클러스티링 기능이 꼭 지원되었으면 합니다.
    화면에 표시할 데이터가 3천개, 1만개가 될 경우에는 커스텀 마커로 표시할 때 보기가 어렵습니다.
2. 매뉴얼에 커스텀 마커에 대한 샘플 예제가 없어 어떻게 활용하는지 모르겠습니다.
3. onMapReady 할 때 GPS 위치좌표를 인식하여 지도 화면이 GPS 현재 위치로 자동 표시되도록 하는 걸 찾는데 쉽지 않네요.
   매뉴얼에 제공해 주실 수 있을까요?

**Reply by note11g** [reply]
1. 클러스터링 기능은 다음 버전에서 지원될 예정입니다.
2. 문서 역시 지금 개편을 준비하고 있습니다. 더 세부적으로 내용이 안내될 수 있도록 노력하겠습니다.
